src/applications/simple-forms/mock-simple-forms-patterns/tests/e2e/mock-simple-forms-patterns.cypress.spec.js

```diff
@@ -99,272 +48,9 @@ const testConfig = createTestConfig(
-              data.exampleArrayOne[0].facilityName,
-            );
-            fillDateWebComponentPattern(
-              'exampleArrayOne_0_from',
-              data.exampleArrayOne[0].from,
-            );
-            fillDateWebComponentPattern(
-              'exampleArrayData_0_to',
-              data.exampleArrayData[0].to,
-            );
-            cy.findByText(/add another facility/i, {
-              selector: 'button',
-            }).click();
-            fillTextWebComponent(
-              'exampleArrayOne_1_facilityName',
-              data.exampleArrayOne[1].facilityName,
-            );
-            fillDateWebComponentPattern(
-              'exampleArrayOne_1_from',
-              data.exampleArrayOne[1].from,
-            );
-            fillDateWebComponentPattern(
-              'exampleArrayData_1_to',
-              data.exampleArrayData[1].to,
-            );
-
-            // array example 2
-            fillTextWebComponent(
-              'exampleArrayTwo_0_name',
-              data.exampleArrayTwo[0].name,
-            );
-            selectYesNoWebComponent(
-              'exampleArrayTwo_0_proof',
-              data.exampleArrayTwo[0].proof,
-            );
-
-            cy.axeCheck();
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [pagePaths.arrayMultiplePage]: ({ afterHook }) => {
-        cy.injectAxeThenAxeCheck();
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            // array example 1
-            fillFullNameWebComponentPattern(
-              'exampleArrayData_0_fullName',
-              data.exampleArrayData[0].fullName,
-            );
-
-            cy.findByText(/add another child/i, {
-              selector: 'button',
-            }).click();
-
-            fillFullNameWebComponentPattern(
-              'exampleArrayData_1_fullName',
-              data.exampleArrayData[1].fullName,
-            );
-
-            cy.axeCheck();
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [replaceArrayIndexPath(pagePaths.arrayMultiplePageItem, 0)]: ({
-        afterHook,
-      }) => {
-        cy.injectAxeThenAxeCheck();
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            fillAddressWebComponentPattern(
-              'address',
-              data.exampleArrayData[0].address,
-            );
-
-            cy.axeCheck();
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [replaceArrayIndexPath(pagePaths.arrayMultiplePageItem, 1)]: ({
-        afterHook,
-      }) => {
-        cy.injectAxeThenAxeCheck();
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            fillAddressWebComponentPattern(
-              'address',
-              data.exampleArrayData[1].address,
+              `wcv3Address_state`,
```

Comment:
address pattern still needs to do some manual correction, since it changes to select input after the page has loaded.